engineering-decision-records/005-amaru-maintainers-committee.md

```diff
@@ -0,0 +1,39 @@
+---
+type: process
+status: accepted 
+---
+
+# Amaru maintainers committee composition
+
+## Context
+
+Amaru is meant to be(come) a rather large and complex project, spanning over multiple years. 
+
+The ambition of the project is to build a new fully interoperable block-producing node for improving the overall performance the Cardano blockchain. 
+
+The Amaru node wants to provide a simplified entry point for building things on Cardano by using a modular design and Rust as its main coding language.
+
+## Motivation
+
+In order to manage the decision making on the project and have a compelling product oversight, Matthias and Santiago want to extend their maintainers committee's responsibilities to other members.
+
+## Decision
+
+Here is the new composition of the Amaru maintainers committee:
+
+| Matthias Benkort <br/> [@KtorZ][] | Santiago Carmuega <br/> [@scarmuega][] |Andrew Westberg <br/> [@AndrewWestberg][] |  Arnaud Bailly <br/> [@abailly][] | Chris Gianelloni <br/> [@wolf31o2][] |   Pi Lanningham <br/> [@Quantumplation][] | Sebastien Guillemot <br/> [@wolf31o2][] |
+| ---                               | ---                                           | ---                            | ---                            | ---                             | ---                            | ---                             |
+
+[@KtorZ]: https://github.com/ktorz
+[@scarmuega]: https://github.com/scarmuega
+[@AndrewWestberg]: https://github.com/AndrewWestberg
+[@abailly]: https://github.com/abailly
+[@wolf31o2]: https://github.com/wolf31o2
+[@Quantumplation]: https://github.com/quantumplation
+[@sebastienGllmt]: https://github.com/sebastienGllmt
```

Comment:
Would it make sense to have the general bounded-contexts be reflected here, with their responsible maintainers?